### ISSUE TITLE
Media embed block on horizontal

### ISSUE THREAD

**Issue by allanaaa** [omeka-s-themes/foundation#77]
I've got items in a horizontal media embed, and there is a "not last child" effect that's causing the thumbnails to mis-align.

I think you only want this margin-bottom on vertical, not horizontal?

<img width="1109" alt="Screenshot 2024-11-18 at 13 01 46" src="https://github.com/user-attachments/assets/2ad90bb0-5bab-4c4f-8613-d604fbec174d">

http://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-fourth-site/page/testing-layouts

**Comment by kimisgold** [comment]
This is actually an issue stemming from the default styles provided by the core. If you pull latest changes from the core now, there should be a fix in place.

**Comment by allanaaa** [comment]
I'm not seeing any change, sorry. I cleared my cookies and everything! Looked at it in Chrome too.

<img width="893" alt="Screenshot 2024-11-20 at 14 38 02" src="https://github.com/user-attachments/assets/a2e7db0b-eae1-4b42-b25e-3329365a30b0">


I also realized that when logged-out the page is showing an error about an item that can't be found - this appears to be an issue with Hierarchy, so I'll make an issue there - just wanted to mention it if you tried to see that page but couldn't.
